### DISCUSSION TITLE
트랜잭션의 범위 최소화의 의미

### DISCUSSION THREAD

**Discussion by choihuk** [Learning-Is-Vital-In-Development/23-6-RealMySQL#22] [Q&A]
158p: 트랜잭션의 범위를 최소화하라는 뜻이 @transactional(propagation = Propagation.REQUIRES_NEW) 옵션으로 도메인에 맞게 각각의 물리 트랜잭션으로 나누라는 뜻인가요?

**Comment by dojinyou** [comment]
논리적으로 분리할 수 있는 트랜잭션이라면 가능하면 물리적으로 트랜잭션을 분리하는 게 가용성과 정합성 측면에서 더 좋은 것 같아서 그렇게 이야기하신 것 같아요!
